### ISSUE TITLE
QA Ph4: Accessibility_Linked images missing alternative text 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#908]
Using the WAVE extension for Chrome with the filter set to WCAG 2.0 AA, there are 20 errors for linked images missing alternative text, all are found in the live modal on the social tab. 

Steps: 
1. Go to https://io-webapp-staging.appspot.com/io2016/
2. Enable the WAVE extension
3. View style errors with filter set to WCAG 2.0 AA 
4. Note linked image missing alternative text errors 

Chrome 50 Mac OS X 10.11  

![gio_908](https://cloud.githubusercontent.com/assets/7585604/15232164/39ce652a-1854-11e6-8385-cffcdc5deb0d.png)
